streamer/src/nonblocking/quic.rs

```diff
@@ -1199,6 +1184,11 @@ async fn handle_connection(
 
         stats.total_streams.fetch_sub(1, Ordering::Relaxed);
         stream_load_ema.update_ema_if_needed();
+
+        let new_window = compute_receive_window_bdp(max_receive_rate_kbps, connection.rtt());
```

Comment:
and we re-compute it after a full stream is received to keep it in line with what is going on in the channel